ci/integration/logging/run_sg.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/env bash
+# CI script for CentOS8 job
+# purpose: spawn sg-core to process messages sent by rsyslog
+
+set -ex
+
+# enable required repo(s)
+yum install -y epel-release
+
+# Without glibc-langpack-en locale setting in CentOS8 is broken without this package
+yum install -y git golang gcc make glibc-langpack-en qpid-proton-c-devel
+
+export GOBIN=$GOPATH/bin
+export PATH=$PATH:$GOBIN
+
+SOCKET=/tmp/rsyslog-test-socket
+BRIDGE_LOG=/var/log/sg-bridge.log
+
+# install and start sg-bridge
+git clone https://github.com/infrawatch/sg-bridge.git
```

Comment:
Cloning and building this each time seems a bit heavy no? Is there a dependency here where you need to build sg-bridge? If there is a cross-project interface here, I would like to see it try and clone the same-named branch like we do for STF CI stuff, with a fallback to `master` branch.

Reply:
I think the simplest thing is to basically check if the same-named branch exists on the sg-bridge side (for coordinated testing if changes on sg-bridge are required to satisfy a change here), and if not, then just pull from quay.io/infrawatch/sg-bridge:latest

It's not that big of a deal, but it could certainly be a bit faster test, although I suspect sg-bridge doesn't ultimately take all that long to deploy.

If you want to make the changes that's fine, if not, I won't block it.

Reply:
So ... you want me to run sg-bridge in separate container instead of sg-core container and in case the sg-bridge repo does not have the same topic branch use the prebuilt image ... got it.